app/db.py

```diff
@@ -30,8 +30,28 @@ class CaseObject(BaseModel):
     outcome: Optional[str] = "Granted"
     country_of_origin: Optional[str] = "Kazakhstan"
     pdf_file: Optional[str] = 'C://file/some/where'
```

Comment:
is this hard coding example of the path temporary? Since we seem to use this same file path in a lot of different files, what do you think about using a string like 'file_path' instead so the next team only has to change it in one place - it can be imported to the files it's used?

Reply:
can do. and I made that change.